roles/lib/files/FWO.Data/Role.cs

```diff
@@ -20,12 +21,33 @@ public Role(Role role)
             Users = new (role.Users);
         }
 
+        private static string LdapDnExtractName(string dn, string name)
```

Comment:
how about new DistName(dn).UserName ?
Don't understand the Replace("\\2c"...

Reply:
good idea - implemented this